extensions/amp-story/0.1/progress-bar.js

```diff
@@ -69,24 +73,26 @@ export class ProgressBar {
   }
 
   /**
-   * @param {number} pageCount The number of pages in the story.
+   * @param {!Array<string>} segmentIds The id of each page in the story.
    * @return {!Element}
    */
-  build(pageCount) {
+  build(segmentIds) {
```

Comment:
was your intention to rename all `page` to `segment` ?

I saw now it's a mix in this file.

For example, it's called segmentId, but then pageId in a later place.

Reply:
see above